frescobaldi_app/vcs/gitdoc.py

```diff
@@ -298,6 +298,7 @@ def set_status(gitprocess, exitcode):
 
         # arguments to get file status
         args = [
+            # TODO: really? including ignored files?
```

Comment:
At that time I was think that ignored files are a part of the repository and users may open them in Frescobaldi.    

This will also help us deal with the ignored files. Since we just judge a file through its location. If we check this file's status at first, then we don't need to handle the possible errors brought by tracking a git ignored file's diff result.